cri/v1alpha2/service/cri.go

```diff
@@ -19,11 +20,12 @@ type Service struct {
 
 // NewService creates a brand new cri service.
 func NewService(cfg *config.Config, criMgr cri.CriMgr) (*Service, error) {
+	unaryInterceptor := middleware.HandleWithGlobalMiddlewares(metrics.GRPCMetrics.UnaryServerInterceptor())
 	s := &Service{
 		config: cfg,
 		server: grpc.NewServer(
 			grpc.StreamInterceptor(metrics.GRPCMetrics.StreamServerInterceptor()),
-			grpc.UnaryInterceptor(metrics.GRPCMetrics.UnaryServerInterceptor()),
+			grpc.UnaryInterceptor(unaryInterceptor),
```

Comment:
This PR's works are covered by #2613 in some way.  Besides, using `grpc_middleware.WithUnaryServerChain` to link each middleware is a more general way. 😄 Could you help me review this PR #2613 ? You can make some changes based on my works.